.buildkite/build.rayci.yml

```diff
@@ -0,0 +1,39 @@
+group: build
+steps:
+  - label: ":tapioca: build: wheel {{matrix}}"
+    instance_type: medium
+    commands:
+      - ./ci/build/build-manylinux-ray.sh
+      - ./ci/build/build-manylinux-wheel.sh {{matrix}}
+      - if [ "$BUILDKITE_PULL_REQUEST" != "false" ]; then exit 0; fi
```

Comment:
why early exit?

Reply:
the copy logic down below should not be run on a pull request; also a feature parity with existing logic